com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -765,7 +779,8 @@ protected static ClearFlag GetCameraClearFlag(ref CameraData cameraData)
                 return (cameraData.clearDepth) ? ClearFlag.DepthStencil : ClearFlag.None;
 
             // Always clear on first render pass in mobile as it's same perf of DontCare and avoid tile clearing issues.
-            if (Application.isMobilePlatform)
+            if(Application.isMobilePlatform ||
+               (!cameraData.isPreviewCamera && DebugHandler != null && DebugHandler.IsScreenClearNeeded))
```

Comment:
Would be nice to add comment explaining this logic.

Reply:
Comment added.